package.json

```diff
@@ -467,6 +467,7 @@
 	},
 	"scripts": {
 		"vscode:prepublish": "npm run -S esbuild-base -- --minify",
+		"postinstall": "npx npm-license-crawler --onlyDirectDependencies --csv third-party-licenses.csv",
```

Comment:
This makes it so that any time we do a `yarn add`, it regenerates third-party licenses csv